packages/grafana-data/src/types/datasource.ts

```diff
@@ -580,13 +581,6 @@ export interface QueryFix {
   action?: QueryFixAction;
 }
 
-export interface QueryFixAction {
-  type: string;
-  query?: string;
-  preventSubmit?: boolean;
-  options?: KeyValue<string>;
-}
-
```

Comment:
What's the reasoning to move this to `./logs`? I also see other non-logging datasource use it, e.g. Prometheus.

Reply:
Hm, I think I understand the reasoning. But question is if we should do this? I rather think we can move `QueryFixType` to `./datasource.ts` as well?

Reply:
>  I also see other non-logging datasource use it, e.g. Prometheus.

Good point.

Reply:
Done.